database/factories/SubscriptionFactory.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\Customer;
+use App\Models\Package;
+use App\Models\Plan;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\Subscription>
+ */
+class SubscriptionFactory extends Factory
+{
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition(): array
+    {
+        $plan = Plan::factory()->create();
+        $package = Package::factory()->create();
+        $customer = Customer::factory()->create();
+
+        return [
+            'plan_id' => $plan->id,
+            'package_id' => $package->id,
+            'customer_id' => $customer->id,
```

Comment:
For laravel factories that creates another relation we usually avoid calling ->create, we just pass the factory instance